doc/sphinx/proof-engine/tactics.rst

```diff
@@ -479,6 +479,7 @@ one or more of its hypotheses.
    .. prodn::
       occurrences ::= at @occs_nums
       | in @goal_occurrences
+      simple_occurrences ::= @rewrite_occs
```

Comment:
Good catch.  That error crept in because `occurrences` in the mlgs maps to `rewrite_occs`; `clause_dft_concl` maps to `occurrences`.  I'll do another round of nonterminal name cleanup at some point.